build_scripts/ci/e2e_tests.sh

```diff
@@ -0,0 +1,23 @@
+set -eu
+
+API_KEY=$1
+RED='\033[0;31m'
+NC='\033[0m'
+
+echo "$API_KEY" > .test-token
+
+start_time=$(date +%s%3N)
+set +e
+make e2e-test
+exit_code=$?
+set -e
+end_time=$(date +%s%3N)
+
+echo ""
+echo -e "${RED}Logs${NC}: https://app.datadoghq.eu/logs?from_ts=$start_time&to_ts=$end_time&index=%2A&live=false&query=stack%3Aapp"
```

Comment:
nit: Why do we use red here?
These get printed no matter what the exit code is, so maybe this should be a neutral colour instead, like blue?

Reply:
As we discussed in person, changed the colour to red on failure and green on success.